### ISSUE TITLE
Question about "rf" parameter in conv1d() function

### ISSUE THREAD

**Issue by XisnZhang** [openai/finetune-transformer-lm#31]
if rf == 1:  # faster 1x1 conv
           c = tf.reshape(tf.matmul(tf.reshape(x, [-1, nx]), tf.reshape(w, [-1, nf])) + b, shape_list(x)[:-1] + [nf])
        else:  # was used to train LM
            c = tf.nn.conv1d(x, w, stride=1, padding=pad) + b  

when training LM, rf = ?  and why?
